skale/dataclasses/current_node_info.py

```diff
@@ -21,11 +21,20 @@
 
 
 class CurrentNodeInfo(NodeInfo):
-    def __init__(self, node_id, node_name, base_port, bind_ip):
+    def __init__(self, node_id, node_name, base_port, bind_ip, ima_mainnet=None,
+                 ima_mp_schain=None, ima_mp_mainnet=None, wallets=None):
         self.bind_ip = bind_ip
+        self.ima_mainnet = ima_mainnet
+        self.ima_mp_schain = ima_mp_schain
+        self.ima_mp_mainnet = ima_mp_mainnet
+        self.wallets = wallets
         super().__init__(node_id, node_name, base_port)
 
     def to_config(self):
         return {**super().to_config(), **{
```

Comment:
The node info may be a little fat, so I think it will be better to create only one dict here. This code creates 2 redundant dict instances.